LoopKitUI/Views/QuantityPicker.swift

```diff
@@ -101,20 +101,32 @@ public struct QuantityPicker: View {
     }
 
     public var body: some View {
-        Picker("Quantity", selection: selectedValue) {
-            ForEach(selectableValues, id: \.self) { value in
-                Text(self.formatter.string(from: value) ?? "\(value)")
-                    .foregroundColor(self.colorForValue(value))
-                    .anchorPreference(key: PickerValueBoundsKey.self, value: .bounds, transform: { [$0] })
-                    .accessibility(identifier: self.formatter.string(from: value) ?? "\(value)")
+        picker
+            .labelsHidden()
+            .pickerStyle(WheelPickerStyle())
+            .overlayPreferenceValue(PickerValueBoundsKey.self, unitLabel(positionedFrom:))
+            .accessibility(identifier: "quantity_picker")
+    }
+
+    @ViewBuilder
+    private var picker: some View {
+        if #available(iOS 15.1, *) {
```

Comment:
@ps2 Looking for your opinion here.  I went back & forth on making this conditional.  I don't really feel strongly either way but here it is for illustrative purposes at least.

Reply:
I believe that if the built-in SwiftUI PickerView supported resizing, it would look like ours does with the fix. Our fix is not a hack like the others trying to deal with the effects of a picker that cannot be resized, because it actually makes the picker resizeable, so all the other expected things, like rounded corners, not rendering out of bounds, accepting hits out of bounds, etc, just work.